CarPG/Assets/Scripts/Damagable.cs

```diff
@@ -11,6 +11,7 @@ public class Damagable : MonoBehaviour
     public float damageMultiplier=1;
     [HideInInspector]
     public Damagable healthPool=null;
+    public GameObject fireball;
```

Comment:
Because Damagable has a fireball object that means all damagable objects get it too. Realistically you should move this to PlayerDeath.cs because not all objects that can be killed explode on death